lib/translit.rb

```diff
@@ -32,10 +31,21 @@ def self.convert(text, enforce_language = nil)
 
   def self.invert_character_map(map)
     map.dup.inject({}) do |acc, tuple|
+      rus = tuple.last.first
+      eng_value       = tuple.first
+      acc[rus] ? acc[rus] << eng_value            : acc[rus] = [eng_value]
+      acc
+    end
+  end
```

Comment:
This function is now simplified, because it just needs to flip latin and cyrillic characters in the hash. No need to deal with capitalization since the hash will have already been run through the latin_cases method below.